src/test/java/net/helpscout/api/ApiClientTest.java

```diff
@@ -1,12 +1,26 @@
 package net.helpscout.api;
 
-import static com.github.tomakehurst.wiremock.client.WireMock.*;
```

Comment:
Usually I prefer to be explicit, perhaps we need to open this one up to a team vote?